stock/templates/public/base.html

```diff
@@ -111,12 +111,6 @@
                         </li>
 
                         <li>
-                            <a class="{% navactive request 'auth_api_key_management auth_add_api_key' %}" href="{% url 'auth_api_key_management' %}">
-                                <i class="fa fa-key fa-fw grayiconecolor"></i>{% trans " Api Keys" %}
-                            </a>
-                        </li>
-
-                        <li>
                             <a class="{% navactive request 'auth_characters' %}" href="{% url 'auth_characters' %}">
```

Comment:
I would also delete this navbar link. The only purpose this page serves is to set a main character which can be reached from the dashboard.